src/sass/components/_toasts.scss

```diff
@@ -19,12 +27,37 @@
 .toast {
   max-width: 100%;
   width: 100%;
+  font-size: $font-size-base * 1.125;
 }
 
 .toast-body {
   display: flex;
 }
 
+.toast-icon-wrapper {
+  display: inline-flex;
+  align-items: center;
```

Comment:
Moet dat eigenlijk niet alfabetisch staan volgens de stijlregels?

Reply:
Daar is niet echt een vaste stijlregel voor, sommige mensen doen het alfabetisch maar ik groepeer volgens type, font styles samen, en in dit geval eerst display type en dan de specificaties voor flex. Dat vind ik logischer dan het alfabetisch te doen, zo kan je snel alles wat bij elkaar hoort samen zien.

Reply:
Euh, hebben we geen CI job die dat checkt? Vb: https://git.sumocoders.be/deboeververzekeringen/ivecoinsurancebe/-/jobs/275373. Ale, ik wil gewoon zeker zijn dat het de stijlregels volgt, en dus niet in alle projecten gaat failen.

Reply:
Ok. Eventueel lokaal in pre-commit hook steken om stylelint te runnen?
Ik dacht dat dat automatisch gebeurde voor nieuwe projecten?